config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resources :articles, except: [:edit, :update]
+  resources :comments, only: [:create] do
```

Comment:
I _think_ the `only` here can be an empty Array. it's unclear to me why there are 3 ways to create a Comment though. Would you be able to elaborate?

Reply:
I had three ways for comments because a comment could be created from an article or a comment. However, the original `comments_path` is unused so I altered my code so there's no need for nested resources.